src/backend/tests/change-requests.test.ts

```diff
@@ -289,6 +289,71 @@ describe('Change Requests', () => {
     });
   });
 
+  describe('add Proposed Solution', () => {
```

Comment:
looks like some mixing here

Reply:
I dont know what github is doing tbh, this is weird